.github/workflows/lint-test.yaml

```diff
@@ -142,7 +142,7 @@ jobs:
           fetch-depth: 0 # get all the history because cargo xtest --change-since origin/main requires it.
       - uses: ./.github/actions/rust-setup
       - run: docker run --detach -p 5432:5432 cimg/postgres:14.2
-      - run: cargo nextest --nextest-profile ci --test-threads 16 --retries 2 --unit --exclude backup-cli --changed-since "origin/main"
```

Comment:
also removing the retries and test-threads parameters since it appears this testsuite is actually not flaky anymore these days :)

Reply:
😱 This feels risky to me...

Reply:
🙏.

If we see it flaking again I guess we can add the retries back but I'd prefer to not limit the test threads since it also makes the test faster that way.